test/integration/clinicians/clinician-sidebar.js

```diff
@@ -1,24 +1,22 @@
+import { v4 as uuid } from 'uuid';
+
 import { getErrors, getRelationship } from 'helpers/json-api';
 
-import { testTs } from 'helpers/test-timestamp';
 import stateColors from 'helpers/state-colors';
 
 import { teamCoordinator, teamNurse } from 'support/api/teams';
 import { roleAdmin, roleEmployee, roleManager } from 'support/api/roles';
-import { getWorkspaces, workspaceOne } from 'support/api/workspaces';
+import { workspaceOne } from 'support/api/workspaces';
 import { getClinician, getCurrentClinician } from 'support/api/clinicians';
 
 const testClinician = getClinician({
-  id: '1',
+  id: uuid(),
   attributes: {
     name: 'Test Clinician',
     email: 'test.clinician@roundingwell.com',
-    last_active_at: testTs(),
-    enabled: true,
```

Comment:
`enabled: true` is set for each clinician fixture, so we don't need to set it in `getClinician()`.